### ISSUE TITLE
uni model: stuck on a bug

### ISSUE THREAD

**Issue by matsc-at-sics-se** [matsc-at-sics-se/unison#24]
I just can't find this one.
Tracing suggests that there is some problem with `InfiniteRegisterClass M256`, which I added today.
It seems to happen when trying to create that entry of the `atoms` JSON array.
Infinite register classes up to `M128` work fine.
```
matsc@teterev:~/unison/experiments/x86/fft-from-kff$ uni model mk_twiddles.alt.uni 
uni: Map.!: given key is not an element in the map
```
[mk_twiddles.alt.uni.txt](https://github.com/matsc-at-sics-se/unison/files/1943593/mk_twiddles.alt.uni.txt)

**Comment by matsc-at-sics-se** [comment]
Included `GR32orGR64` in `regClasses` by commit `9172906dfa3d5e24a8b28ef7669d424a33b46222`.
No crash no more, but `uni model` outputs _nothing_!?

**Comment by matsc-at-sics-se** [comment]
Wrong, the bug is still there:
```
uni: Map.!: given key is not an element in the map
```
I'm still stuck.

**Comment by matsc-at-sics-se** [comment]
Found it by chance.  First, the error could be narrowed down to here:
```
wRegAtoms raf r2a r =
    let (fs, ls) = mapTuple ((M.!) r2a) (raf r)
    in [minimum fs..maximum ls]
```
with a key `m10399` not in the map.  On a hunch, I changed `16` to `32` here:
```
roundToRc n =
    let n' = toRational n
        w  = 32
    in w * (ceiling (n' / (toRational w)))
```

But `uni model` is still fishy. With 559 operations, `uni model` takes 47 seconds.
With maximum 153 stores and 5 rematerializable operations, the register array contains 10400 (325 x 32) register atoms in "memory" and 10400 (325 x 32) register atoms in "virtual memory".
This seems excessive.
